org.lflang/src/org/lflang/generator/ts/TSGenerator.kt

```diff
@@ -242,16 +240,10 @@ class TSGenerator(
      * Generate the code corresponding to [federate], recording any resulting mappings in [codeMaps].
      */
     private fun generateCode(
-        federate: FederateInstance,
         codeMaps: MutableMap<Path, CodeMap>,
         dockerGenerator: TSDockerGenerator
     ) {
         var tsFileName = fileConfig.name
-        // TODO(hokeun): Consider using FedFileConfig when enabling federated execution for TypeScript.
-        // For details, see https://github.com/icyphy/lingua-franca/pull/431#discussion_r676302102
-        if (isFederated) {
-            tsFileName += '_' + federate.name
-        }
```

Comment:
I have a general question regarding "isFederated" blocks that are being removed here. (Not those with FIXMEs)

Where is this (e.g., formatting the shell script's name) going to be handled once fed-gen is done? I guess in FedGenerator?

Reply:
> I guess in FedGenerator?

Yes, that's the plan at least...